.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
+*.sum
+vendor
```

Comment:
could you place this and the .idea into a local gitignore rule as to keep the shared repo-level .gitignore as simple as possible?